examples/asset/texture_sampler.rs

```diff
@@ -0,0 +1,108 @@
+//! By default Bevy loads images to textures with sampler settings that clamp the image to the edges.
+//! This example shows how to change the sampler settings to repeat the image instead.
+
+use bevy::app::App;
+use bevy::app::Startup;
+use bevy::asset::{AssetServer, Assets};
+use bevy::math::Vec3;
+use bevy::prelude::DefaultPlugins;
+use bevy::prelude::OrthographicProjection;
+use bevy::prelude::{Camera2dBundle, ColorMaterial, Commands, Mesh, ResMut, Transform};
+use bevy::render::camera::ScalingMode;
+use bevy::render::mesh::Indices;
+use bevy::render::mesh::PrimitiveTopology;
+use bevy::render::texture::{
+    ImageAddressMode, ImageLoaderSettings, ImageSampler, ImageSamplerDescriptor,
+};
+use bevy::sprite::MaterialMesh2dBundle;
+
+fn mesh() -> Mesh {
```

Comment:
Why not just use the built in `Quad`? Also, this should have a comment explaining what it does or at least why it's important for this example.

Reply:
> Why not just use the built in `Quad`? 

UV coordinates of corners need to be in range `-1..=2` to demonstrate the issue, which built in `Quad` does not allow.

> Also, this should have a comment explaining what it does or at least why it's important for this example.

I'm hesitant to add trivial comments, just repeating code in English, but if you insist, I will.